src/components/Tag.js

```diff
@@ -24,22 +24,22 @@ class Tag extends Component {
       isDragging,
       connectDropTarget,
       readOnly,
+      tag,
     } = props;
-
-    const tagComponent = (
-      <span
-        className={ClassNames('tag-wrapper', props.classNames.tag, {'opacity-none' : isDragging}, {'cursor-move': canDrag(props)})}
-        onClick={props.onTagClicked}
-        onKeyDown={props.onTagClicked}>
-        {label}
-        <RemoveComponent
-          tag={props.tag}
-          className={props.classNames.remove}
-          removeComponent={props.removeComponent}
-          onClick={props.onDelete}
-          readOnly={readOnly}
-        />
-      </span>
+    const { className = '' } = tag;
+    const tagComponent = ( <span
+      className={ClassNames('tag-wrapper', props.classNames.tag, {'opacity-none' : isDragging}, {'cursor-move': canDrag(props)}, className)}
```

Comment:
`tag` is already destructed above so use tag instead of `props.classNames.tag`

Reply:
can you fix the proptype of tag in line 51, add the id, classname and labelfield in proptype